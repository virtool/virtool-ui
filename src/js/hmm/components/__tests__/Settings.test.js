import HMMSettings from "../Settings";

describe("<HMMSettings />", () => {
    it("should render", () => {
        const wrapper = shallow(<HMMSettings />);
        expect(wrapper).toMatchSnapshot();
    });
});
