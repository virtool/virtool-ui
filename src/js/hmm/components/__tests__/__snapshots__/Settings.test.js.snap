// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HMMSettings /> should render 1`] = `
<div>
  <ViewHeader
    title="HMM Settings"
  >
    <ViewHeader__ViewHeaderTitle>
      HMM Settings
    </ViewHeader__ViewHeaderTitle>
  </ViewHeader>
  <Connect(HMMPurge) />
</div>
`;
